code/game/machinery/computer/prisoner.dm

```diff
@@ -7,7 +7,7 @@
 	light_color = LIGHT_COLOR_ORANGE
 	req_access = list(access_armory)
 	circuit = /obj/item/circuitboard/prisoner
-	var/id = 0.0
+	id = 0.0
```

Comment:
```suggestion
```

Bad default value.

Reply:
these are used by computers internally

Reply:
This expects the mapper to have set it. 0.0 is as nonsensical as null. So delete this override, let it be null.